articles/how-to-build-a-music-player-using-django/index.md

```diff
@@ -0,0 +1,373 @@
+Have you ever wanted to build a music player web application?. If your answer is yes then this article is for you. I have always wanted to build a music player and finally, I did. I love how Django's ORM makes it so easy to work with a database along with the views (i.e the functional backend of the app) and the template files all connected with the Django MVT (i.e Model View Template) architecture. Django is a perfect choice when building a music player web application, and I would be happy to walk you through every step of how I achieved it.
+
+The purpose of this article is to introduce the reader to the Django web framework and explore its ORM and MVT capabilities by engaging him/her in making use of Django to build a simple music player for their personal use. Readers are expected to have some experience with HTML, CSS, and Javascript as this article will be focusing majorly on the backend implementation.
+
+If you are new to Django and want to get started with it, you can refer to the [official Django documentation](https://docs.djangoproject.com/en/3.1/).
+
+### Prerequisites
+
+Install python and enable `pip`. Then run the following to install the required packages:
+
+```bash
+pip install django
+pip install pillow
```

Comment:
Briefly explain about the `pillow` package and how you are going to use it in this project.